lighthouse-core/report/v2/renderer/util.js

```diff
@@ -69,6 +80,86 @@ class Util {
     }
     return formatter.format(new Date(date));
   }
+
+  /**
+   * @param {!URL} parsedUrl
+   * @param {{numPathParts: number, preserveQuery: boolean, preserveHost: boolean}=} options
+   * @return {string}
+   */
+  static getDisplayName(parsedUrl, options) {
```

Comment:
can we call this `getUrlDisplayName` now that it's a generic utility? thanks for moving I've been wanting to clean up the `getDisplayName` sprinkled everywhere and now it'll be easy :)